templates/rhpam73-trial-ephemeral.yaml

```diff
@@ -108,6 +108,23 @@ parameters:
   name: BUSINESS_CENTRAL_HOSTNAME_HTTP
   value: ''
   required: false
+## OpenShift Enhancement BEGIN
+- displayName: Enable OpenShift Integration
+  description: If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property)
+  name: KIE_CONTROLLER_OCP_ENABLED
+  value: "false"
+  required: false
+- displayName: Prefer KIE Server OpenShift Service
+  description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+  name: KIE_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE
+  value: "true"
```

Comment:
This is "false" everywhere else. Shouldn't it also be here (instead of "true")?

Reply:
Changed to 'false'. Set to true is to facilitate local OpenShift cluster (aka 'oc cluster up') test, but it is a nice to have feature.